fl4health/clients/fenda_ditto_client.py

```diff
@@ -9,7 +9,7 @@
 from fl4health.checkpointing.client_module import ClientCheckpointModule
 from fl4health.clients.ditto_client import DittoClient
 from fl4health.model_bases.fenda_base import FendaModel
-from fl4health.model_bases.sequential_split_models import SequentiallySplitExchangeBaseModel
+from fl4health.model_bases.sequential_split_models import SequentiallySplitModel
```

Comment:
While it didn't actually change the flow of the example, since it uses a `FullParameterExchanger` this type is misleading, given that it's mean to facilitate only exchanging the feature extraction module.